metisp/workflows/metis/metis_lm_img_wkf.py

```diff
@@ -30,3 +38,56 @@
                     .with_meta_targets([SCIENCE])
                     .build())
 
+basic_reduction_sky = (task('metis_lm_img_basic_reduce_sky')
```

Comment:
Please prefix all these tasks with `lm`, because otherwise they will conflict with the tasks from the other subinstruments.